app/screens/UserProfile.js

```diff
@@ -122,27 +142,43 @@ class UserProfile extends Component {
                                         />
                                     </ConfirmationModal>
                                     <Text style={s.username}>{user.username}</Text>
+                                    {admin_rank_int &&
```

Comment:
When using the `&&` syntax we need to be careful that what we're evaluating is a boolean (otherwise Android crashes),  so for here, it would be:
```suggestion
                                    {!!admin_rank_int &&
```